src/sparsezoo/analytics.py

```diff
@@ -38,10 +38,9 @@ def analytics_disabled():
     """
     :return: True if analytics should be disabled, False otherwise
     """
-    gdpr = is_gdpr_country()
     env_disabled = os.getenv("NM_DISABLE_ANALYTICS")
 
-    return gdpr or env_disabled
+    return env_disabled or is_gdpr_country()
```

Comment:
I feel the original check made sense too, the idea being we always disable analytics for a gdpr country and optionally for others if NM_DEBUG_ANALYTICS is set but am not opposed to this change.

Also removing the variable makes the expression read better, thank you for that!